src/utils/openmc/statepoint.py

```diff
@@ -7,6 +8,9 @@
 import openmc
 from openmc.constants import *
 
+if sys.version > '3':
```

Comment:
If you add both 2.7 and 3.4 to the Python list in the travis.yml file, it will test both languages in parallel.  This is what pyne and cyclus do.  See https://github.com/pyne/pyne/blob/develop/.travis.yml#L2  Should be a one or two line change,